control/xferfcn.py

```diff
@@ -1010,8 +1014,12 @@ def sample(self, Ts, method='zoh', alpha=None):
         if method == "matched":
             return _c2d_matched(self, Ts)
         sys = (self.num[0][0], self.den[0][0])
-        numd, dend, dt = cont2discrete(sys, Ts, method, alpha)
-        return TransferFunction(numd[0, :], dend, dt)
+        if prewarp_frequency is not None:
+            Twarp = 2*np.tan(prewarp_frequency*Ts/2)/prewarp_frequency
+            numd, dend, _ = cont2discrete(sys, Twarp, method, alpha)
+        else:
+            numd, dend, _ = cont2discrete(sys, Ts, method, alpha)
+        return TransferFunction(numd[0, :], dend, Ts)
```

Comment:
This could be shortened to
```
Twarp = Ts if prewarp_frequency is None else 2*np.tan(prewarp_frequency*Ts/2)/prewarp_frequency
numd, dend, _ = cont2discrete(sys, Twarp, method, alpha)
```